File: PRAK404-2210817220011-MeldianaHandayani.c
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
int main(){
    int pprogram;
    float a,b,hasil, c,d, hasilbagi;
    printf("\tpilih program   \n");
    printf("\t1. Penjumlahan  \n");
    printf("\t2. Pengurangan  \n");
    printf("\t3. Perkalian    \n");
    printf("\t4. Pembagian    \n");
    printf("\t5. Exit         \n");

    printf("Masukkan pilihan  :  ");
    scanf("%d", &pprogram);
    switch (pprogram){
        case 1:
        printf("Masukkan Nilai Pertama   :");
        scanf("%f", &a);
        printf("Masukkan Nilai Kedua     :");
        scanf("%f", &b);
        hasil= a+b;
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n",a,b,hasil);
        break;
        case 2:
        printf("Masukkan Nilai Pertama   :");
        scanf("%f", &a);
        printf("Masukkan Nilai Kedua     :");
        scanf("%f", &b);
        hasil= a-b;
        printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", a,b,hasil);
        break;
        case 3:
        printf("Masukkan Nilai Pertama   :");
        scanf("%f", &a);
        printf("Masukkan Nilai Kedua     :");
        scanf("%f", &b);
        hasil= a*b;
        printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", a,b,hasil);
        break;
        case 4:
        printf("Masukkan Nilai Pertama   :");
        scanf("%f", &a);
        printf("Masukkan Nilai Kedua     :");
        scanf("%f", &b);
        hasil= a/b;
        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", a,b,hasil);
        break;
        case 5:
        printf("Terima kasih, telah menggunakan kalkulator MELDIANA HANDAYANI");
        exit(0);
        default:
        printf("Input anda salah, silahkan coba lagi.\n");
        break;

    }
    main();
return 0;
}

File: PRAK404-2210817220011-MeldianaHandayani.py
print("\tpilih program    ")
print("\t1. Penjumlahan   ")
print("\t2. Pengurangan   ")
print("\t3. Perkalian     ")
print("\t4. Pembagian     ")
print("\t5. Exit          ")

print("Masukkan pilihan   ", end='')
pprogram=int(input())
match pprogram:
    case 1:
        print("Masukkan Nilai Pertama   :", end=' ')
        a=float(input())
        print("Masukkan Nilai Kedua     :", end=' ')
        b=float(input())
        hasil=float( a+b)
        print("Hasil penjumlahan antara {:.2f} dengan {:.2f} adalah {:.2f} ". format (a,b,hasil))
    
    case 2:
        print("Masukkan Nilai Pertama   :")
        a=float(input())
        print("Masukkan Nilai Kedua     :")
        b=float(input())
        hasil=float( a-b)
        print("Hasil pengurangan antara {:.2f} dengan {:.2f} adalah {:.2f} ". format (a,b,hasil))
        
    case 3:
        print("Masukkan Nilai Pertama   :")
        a=float(input())
        print("Masukkan Nilai Kedua     :")
        b=float(input())
        hasil=float( a*b)
        print("Hasil perkalian antara {:.2f} dengan {:.2f} adalah {:.2f} ". format (a,b,hasil))
        
    case 4:
        print("Masukkan Nilai Pertama   :")
        a=float(input())
        print("Masukkan Nilai Kedua     :")
        b=float(input())
        hasil=float( a/b)
        print("Hasil pembagian antara {:.2f} dengan {:.2f} adalah {:.2f} ". format (a,b,hasil))
    case 5:
        print("Terima kasih, telah menggunakan kalkulator MELDIANA HANDAYANI")
    case 6:
           print("Input anda salah, silahkan coba lagi. ")
    
    
    

